bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
i know we have some trouble sometimes where these lines was the "main" actor. There we try to emit an event in a callback of a stopped component ... so if we remove them we enable these behavior where stopped and (maybe) lost components can emit events.